.github/workflows/Android-CI-release.yml

```diff
@@ -1,34 +1,54 @@
-name: Release
+name: Changelog vs Release
 
 on:
   push:
     tags:
       - '*'
 
 jobs:
-  build:
-    name: Publish release
+  release:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2.3.4
         with:
           fetch-depth: 0
+
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@main
+        with:
+          configuration: ".github/changelog-configuration.json"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Install JDK ${{ matrix.java_version }}
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
+
       - name: Install Android SDK
         uses: malinskiy/action-android/install-sdk@release/0.1.0
+
       - name: Build project
         run: ./gradlew clean build
         env:
           VERSION: ${{ github.ref }}
-      - run: |
-          assetsAAR=$(find . -name *release.aar | while read -r asset ; do echo "-a $asset" ; done)
-          VERSION=$(echo $VERSION | cut -d'/' -f3)
-          tag_name="${GITHUB_REF##*/}"
-          hub release create ${assetsAAR} -m "$tag_name" "$tag_name"
+
+      - name: Get the version
+        id: tagger
+        uses: jimschubert/query-tag-action@v2
+        with:
+          skip-unshallow: 'true'
+          abbrev: false
+          commit-ish: HEAD
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
+        with:
+          tag_name: ${{ github.ref }}
+          name: ${{ steps.get_version.outputs.VERSION }}
```

Comment:
Almost done, please use 

`name: ${{steps.tagger.outputs.tag}}`

Reply:
Missed, doing it